gyft.py

```diff
@@ -155,8 +158,9 @@ def merge_slots(in_dict):
                 logging.info(" {} - {}".format(course_code, course_name))
     except Exception as e:
         print()
-        logging.error(" Error while scraping course names for department {}".format(dept))
+        logging.error(" Error while scraping course names for course letter code {}".format(dept))
         logging.error(e)
+        logging.error(" If you know the department code corresponding to the above code, please add it in the code after line 139, and submit a PR.")
```

Comment:
I feel like this message could be improved, because first of all line 139 is not gonna remain the same after more additions, so this would have to be changed too. 
I am open to suggestions on better possible messages.